src/status_im2/contexts/onboarding/profiles/view.cljs

```diff
@@ -239,10 +242,12 @@
 
 (defn views
   []
-  (reset! show-profiles? false)
-  (fn []
-    [:<>
-     [background/view true]
-     (if @show-profiles?
-       [profiles-section]
-       [login-section])]))
+  (let [show-profiles?    (reagent/atom false)
```

Comment:
why not in app-db ?

Reply:
It was a global atom that was not required to be a global atom, I mean, it can be a local atom and everything would work.

I'm not a big fan of moving a local state to a global scope, so if we can keep it local, I'd prefer it over  adding more values to app-db.

Reply:
thats fine if local state is really local, and could be used in context of one component, here you have to add additional functions, pass parameters etc, and that makes it more complicated than just adding this to app-db, don't worry about app-db, that's the idea of re-frame

Reply:
Passing callbacks through components I think is ok. For re-frame we need to find a meaningful path, create a subscription to read the value and create an event to update that value, In this case I think it's easy and more simple just passing the callback.

I care about re-frame because when it is too big, it starts to be a little hard to maintain, also is not always easy to name paths and structure the info, so I usually avoid adding info that is not required to be there.

Probably we could discuss it involving more devs? I'm not sure if we already have a consensus. In the meantime I could revert these changes since they are not the main scope of this PR, and I've seen some global atoms for components/screens across the code, so I think it'd be something good to clarify.

wdyt?

Reply:
could we move functions and atom outside view ?